homeassistant/strings.json

```diff
@@ -71,7 +71,9 @@
       "standby": "Standby",
       "paused": "Paused",
       "home": "Home",
-      "not_home": "Away"
+      "not_home": "Away",
+      "charging": "Charging",
```

Comment:
Side note: I think these entries are becoming so numerous that it's better to sort the list :abcd:.

Reply:
Sounds reasonable, will create a PR later 👍

Reply:
should we sort the whole `strings.json`?

Reply:
Sounds good.

Reply:
-> #139300